### PR TITLE
[#297] Add ability for generating entity relationships diagrams

### PR DESCRIPTION
**Pull request by antonfefilov**
Related to: #297

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ vendor/ruby
 ._*
 .env
 .pryrc
+*.swp
```
Comment (timurvafin) [inline .gitignore]: Please ignore them in your local `~/.gitignore`

**Comment by timurvafin** [comment]
Is there any example how to use it?

**Comment by antonfefilov** [comment]
@timurvafin where we should put such kind of information?

**Comment by timurvafin** [comment]
@antonfefilov let's keep it as is. 
@vast comments?

**Comment by vast** [comment]
@antonfefilov, please sync your changes with the latest `master` and :shipit:

**Comment by antonfefilov** [comment]
@timurvafin @vast synchronized

**Comment by timurvafin** [comment]
@antonfefilov could you please cleanup/squash commits?

**Comment by timurvafin** [comment]
Merged.

**Pull request closed without merging**